trimmerdecorator.rb

```diff
@@ -0,0 +1,9 @@
+require_relative 'decorator'
+
+# Trims the name string if it's greater than 10 characters long
+class TrimmerDecorator < Decorator
+  def correct_name
+    original_name = @nameable.correct_name
+    original_name.length > 10 ? original_name[0, 10] : original_name
+  end
```

Comment:
- [ ] Since this is an array index, it should start from 0 to 9 if we want to get 10, so I will encourage you to replace the second 10 with 9.